src/Language/PureScript/Make.hs

```diff
@@ -188,6 +188,20 @@ make ma@MakeActions{..} ms = do
   return externs
 
   where
+  checkModuleNames :: m ()
+  checkModuleNames = checkNoPrim *> checkModuleNamesAreUnique
+
+  checkNoPrim :: m ()
+  checkNoPrim =
+    for_ ms $ \m -> do
+      case getModuleName m of
+        mn@(ModuleName (ProperName "Prim" : _)) ->
+          throwError
+            . errorMessage'' (pure (getModuleSourceSpan m))
```

Comment:
You can just use `errorMessage'` and drop the `pure` here (these names are terrible, once we drop unpositioned errors entirely I'll give them better names :wink:)